website/templates/emails/pending_embargo_admin.html.mako

```diff
@@ -6,22 +6,36 @@
     Hello ${user.fullname},<br>
     <br>
     % if is_initiator:
-    You initiated an embargoed registration of ${project_name}. The proposed registration can be viewed here: ${registration_link}.<br>
+    You initiated an embargoed registration of ${project_name} with embargo end date ${embargo_end_date.date()}.
```

Comment:
This language was not marked as changed for non-moderated registries.

Reply:
I'mma guess that this is okay to have for both, since it's unrelated to to moderation. Should we check?